### ISSUE TITLE
this repo is significantly out of date

### ISSUE THREAD

**Issue by zancas** [tower-rs/examples#4]
It doesn't build and it's non-trivial to upgrade it.   I recommend archiving it, as it might be an attractive nuisance to new comers!

**Comment by LucioFranco** [comment]
Yeah, we should clean this up, we need to finish our spring cleaning first in the main repo.

**Comment by zancas** [comment]
Maybe instead of blocking on the Spring Cleaning, it would be best to "archive" this repo, _unti_ Spring Cleaning is done.

My concern is that the project may be losing mindshare because developers land here first..   see that it doesn't `cargo build` and leave, never-to-return.

If it's archived (even temporarily) that provides a clear signal that it shouldn't be expected to "just work".
I don't see much of a cost to this "archiving" strategy, but I can _imagine_ a significant cost to the status quo.

**Comment by LucioFranco** [comment]
Agreed, I will likely move things into the tokio org pretty soon anyways and archive the rest.
